pkg/apis/extensions/register.go

```diff
@@ -52,8 +52,6 @@ func addKnownTypes(scheme *runtime.Scheme) {
 		&DeploymentRollback{},
 		&HorizontalPodAutoscaler{},
 		&HorizontalPodAutoscalerList{},
-		&Job{},
```

Comment:
Right, stupid me. With previous version we had the extensions internals registered in batch internals :/ Thanks, updating.